scripts/create-instance.sh

```diff
@@ -122,4 +122,5 @@ CI=true npm install
 # sphinxdoc-collect-and-build-assets-begin
 ${INVENIO_WEB_INSTANCE} collect -v
 ${INVENIO_WEB_INSTANCE} assets build
+${INVENIO_WEB_INSTANCE} files location local var/data --default
```

Comment:
This is not related to asset building, and the command would fail at the build time if the DB host is not up.  We need to move this for later...